### ISSUE TITLE
Is Select2 considered WCAG compliant?

### ISSUE THREAD

**Issue by GarthMartin** [wet-boew/wet-boew#6784]
I know it is not part of WET, but is Select2 type functionality considered WCAG compliant?

https://select2.github.io/examples.html

Thanks,

Garth

**Comment by LaurentGoderre** [comment]
To know if a plugin is compliant you need to evaluate if against WCAG.

**Comment by neoinsight** [comment]
@GarthMartin If  Select2 does pass a WCAG evaluation and you decide to implement in a prototype, can you let me know, as we may then want to include it in some future usability testing. Thanks.

**Comment by hsrudnicki** [comment]
@GarthMartin @neoinsight @LaurentGoderre : Can this issue be closed?

**Comment by hsrudnicki** [comment]
Made "Going Stale" to close this issue due to no further status update(s)/discussion or if otherwise?
Either way, please update this issue's status, thank you

**Comment by MJAlonzo** [comment]
Found the following article if anyone's still interested: https://make.wordpress.org/accessibility/2015/09/07/accessibility-usertest-select2/

**Comment by nschonni** [comment]
@MJAlonzo that's a useful breakdown that shows that that plug-in really doesn't meet accessibility needs.
